test/trento/sap_systems/sap_system_test.exs

```diff
@@ -1060,6 +1075,55 @@ defmodule Trento.SapSystems.SapSystemTest do
         end
       )
     end
+
+    test "should change the health of the SAP system when its database health has changed" do
+      sap_system_id = Faker.UUID.v4()
+      sid = fake_sid()
+
+      initial_events = [
+        build(:application_instance_registered_event,
+          sap_system_id: sap_system_id,
+          sid: sid,
+          features: "MESSAGESERVER"
+        ),
+        build(:application_instance_registered_event,
+          sap_system_id: sap_system_id,
+          sid: sid,
+          features: "ABAP"
+        ),
+        build(:sap_system_registered_event,
+          sap_system_id: sap_system_id,
+          sid: sid
+        )
+      ]
+
+      assert_events_and_state(
+        initial_events,
+        [
+          UpdateDatabaseHealth.new!(%{
```

Comment:
I'm probably missing something but why do we have here two identical UpdateDatabaseHealth to critical?

Reply:
To see that the 2nd doesn't trigger a new event, as the health is the same